group_job.go

```
@@ -32,69 +39,108 @@
 
-	if err != nil {
-		tempCh := make(chan Result[R], 1)
-		close(tempCh)
-		return tempCh, err
+func (gj *groupJob[T]) close() error {
+	if err := gj.isCloseable(); err != nil {
+		return err
 	}
 
-	return ch, nil
+	gj.ack()
```

golangci-lint (1.64.8)

Error return value of `gj.ack` is not checked

(errcheck)